pkg/kubectl/resource_printer.go

```diff
@@ -515,6 +517,19 @@ var (
 	clusterColumns                   = []string{"NAME", "STATUS", "AGE"}
 	networkPolicyColumns             = []string{"NAME", "POD-SELECTOR", "AGE"}
 	certificateSigningRequestColumns = []string{"NAME", "AGE", "REQUESTOR", "CONDITION"}
+
+	//this is columns for -o wide
```

Comment:
I would group these by object, so you can see the normal and wide columns for each type together